### ISSUE TITLE
4112 is not a valid InputSource when using Mini Displayport

### ISSUE THREAD

**Issue by cryocaustik** [newAM/monitorcontrol#59]
OS: Win10 x64 20H2 (19042.867)
Python: 3.9.1
Package version: 2.4.0

When running `get_input_source` while using mini displayport, it throws an error of `invalid InputSource`

Code:
```python
from monitorcontrol import get_monitors

monitors = get_monitors()
for monitor in monitors:
    with monitor:
        print(monitor.get_input_source())
```

Error
```
Traceback (most recent call last):
  File "c:\dev\lazymonitors\lazy_monitors.py", line 6, in <module>
    print(monitor.get_input_source())
  File "C:\dev\lazymonitors\.venv\lib\site-packages\monitorcontrol\monitorcontrol.py", line 361, in get_input_source
    return InputSource(value).name
  File "c:\users\cryocaustik\appdata\local\programs\python\python39\lib\enum.py", line 315, in __call__
    return cls.__new__(cls, value)
  File "c:\users\cryocaustik\appdata\local\programs\python\python39\lib\enum.py", line 611, in __new__
    raise ve_exc
ValueError: 4112 is not a valid InputSource
```

**Comment by newAM** [comment]
enum in question:

https://github.com/newAM/monitorcontrol/blob/65256e819463f3821b7e2fad43f624836ff833dc/monitorcontrol/monitorcontrol.py#L24-L45

What input source were you using on your monitor? (HDMI, DisplayPort, VGA, ect.)

The MCCS v2.2 says this value is "reserved".
Current guess: `4112` is `0x1010` in hex, somewhere the low byte (`0x10`) got duplicated, and the high byte was not masked.  This theory only makes sense if you are using display port though.

**Comment by cryocaustik** [comment]
I am using mini display port (mDP), which should be translating into 0x10

**Comment by newAM** [comment]
Oh, my bad, it is in the title, sorry.
I guess my theory works out then - I'll mask the value to one byte since the high byte is reserved by spec.  I should have a fix tomorrow, but this weekend at the latest.

**Comment by newAM** [comment]
Added a unit test for this issue and made the fix, should be working in `2.4.1`.

**Comment by cryocaustik** [comment]
Confirmed issue is resolved in 2.4.1; thanks!
